### PR TITLE
add quetz-client

### PR DESCRIPTION
**Pull request by mariobuikhuizen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/quetz-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - requests >=2,<3
+    - conda-verify >=3,<4
+    - conda-build >=3,<4
+
+test:
+  imports:
+    - quetz_client
+
+about:
+  home: http://thesnakepit/quetz
```
Comment (wolfv) [inline recipes/quetz-client/meta.yaml:36]: I don't think this URL would work :)

File: recipes/quetz-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - python >=3.6
+    - requests >=2,<3
+    - conda-verify >=3,<4
+    - conda-build >=3,<4
+
+test:
+  imports:
+    - quetz_client
+
```
Comment (wolfv) [inline recipes/quetz-client/meta.yaml:34]: you can also add a command-test, e.g. executing

```
quetz-client -h
```


by adding 

```
commands:
  - quetz-client -h
```

**Comment by wolfv** [comment]
Also I'm happy to be a maintainer here!

File: recipes/quetz-client/meta.yaml (hunk 1 above)
Comment (mariobuikhuizen) [reply recipes/quetz-client/meta.yaml:36]: good catch :)

File: recipes/quetz-client/meta.yaml (hunk 2 above)
Comment (mariobuikhuizen) [reply recipes/quetz-client/meta.yaml:34]: Adding it, thanks.

**Pull request merged**